app/src/common/css/variables/newColors.scss

```diff
@@ -54,3 +54,4 @@ $COLOR--tag-value-text: #394db6;
 $COLOR--tag-value-background: #ced8fc;
 $COLOR--tag-key-text: #6f4599;
 $COLOR--tag-key-background: #dac3e6;
+$COLOR--settings-menu-hover: #e5e5e5;
```

Comment:
Please remove it and set on hover `$COLOR--bg-100` color the same as for the active item.
Looks like you have some misunderstandings with UX